.travis.yml

```diff
@@ -1,4 +1,7 @@
 language: ruby
 rvm:
+ - 2.1
  - 2.2
+ - 2.3.3
```

Comment:
As you can see here, we need to specify patch version to install Ruby 2.3 or 2.4

https://travis-ci.org/toshimaru/amazon-ecs/builds/214137848

Otherwise, we get an error:

```
Requested binary installation but no rubies are available to download,
```